.travis.yml

```diff
@@ -1,16 +1,26 @@
 os: linux
 language: rust
 cache: cargo
+before_cache:
+  - find target/debug -type f -maxdepth 1 -delete
+  - rm -rf target/.rustc_info.json
+  - rm -rf target/debug/examples/
+  - find target/tests/target/debug/ -name "clap*" -exec rm -rf {} +
+  - find target/tests/target/debug/ -name "trybuild*" -exec rm -rf {} +
+  - find target/debug/* -name "clap*" -exec rm -rf {} +
+  - ls -1 examples/ | sed -e 's/\.rs$//' | xargs -I "{{}}" find target/debug/* -name "{{}}*" -exec rm -rf {} +
+  - ls -1 clap_derive/examples/ | sed -e 's/\.rs$//' | xargs -I "{{}}" find target/debug/* -name "{{}}*" -exec rm -rf {} +
+  - ls -1 tests/ | sed -e 's/\.rs$//' | xargs -I "{{}}" find target/debug/* -name "{{}}*" -exec rm -rf {} +
+  - ls -1 clap_derive/tests/ | sed -e 's/\.rs$//' | xargs -I "{{}}" find target/debug/* -name "{{}}*" -exec rm -rf {} +
```

Comment:
So, I finally figured out the proper caching of built deps in a workspace. Had to remove things in many places. @Dylan-DPC Do we have any rust tools regarding built files managing? You are the @rust-lang expert :)